app/models/concerns/admin_column.rb

```diff
@@ -6,11 +6,25 @@ class << self
       def admin_columns(exclude: nil, include: nil)
         @excluded_admin_columns = exclude || @excluded_admin_columns
         @included_admin_columns = include || @included_admin_columns
+        sorted_columns
+      end
+
+      # Ensure prominence_reason immediately follows prominence
+      def sorted_columns
+        return all_columns unless prominenceable_admin_columns?
+        index = all_columns.index('prominence') + 1
+        all_columns.insert(index, all_columns.delete('prominence_reason'))
```

Comment:
This keeps the original 'prominence_reason' in the array so the column is now displayed twice. Will have to update to delete the duplicate.

Reply:
I thought `delete` removed it? Maybe because we're doing it in one shot?

Reply:
Calling `delete` on `all_columns` doesn't because it's a method which builds/returns a different array instance.